en/README.md

```diff
@@ -0,0 +1,1138 @@
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|procedures|String255 array|M|-|Usual procedures list of sender.|
+|name      |String255    |M|-|Process name of sender. It is used in the log message and etc.|
+
+```json
+{
+  "id": 1,
+  "params": {
+    "name": "exampleName",
+    "procedures": [
+      "getMonitoringServerInfo",
+      "getLastInfo",
+      "putItems",
+      "putArmInfo",
+      "fetchItems"
+    ]
+  },
+  "method": "exchangeProfile",
+  "jsonrpc": "2.0"
+}
+```
+
+***Result(result)***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|procedures|String255 array|M|-|Usual procedures list of destination.|
+|name      |String255    |M|-|Process name of destination. It is used in the log message and etc.|
+
+```json
+{
+  "id": 1,
+  "result": {
+    "name": "exampleName",
+    "procedures": [
+      "getMonitoringServerInfo",
+      "getLastInfo",
+      "putItems",
+      "putHistory",
+      "putHosts",
+      "putHostGroups",
+      "putHostGroupMembership",
+      "putTriggers",
+      "putEvents",
+      "putHostParent",
+      "putArmInfo"
+    ]
+  },
+  "jsonrpc": 2
+}
+```
+
+### getMonitoringServerInfo(method)
+
+To inquire myself conecting information and polling interval to Hatohol server per polling interval is the standard motion, but can inquire anytime.
+
+
+***Request(params)***
+
+ getMonitoringServerInfo method does not have arguments. Send a params of request as empty string.
+
+```json
+{
+  "id": 1,
+  "params": "",
+  "method": "getMonitoringServerInfo",
+  "jsonrpc": "2.0"
+}
+```
+
+***Result(result)***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|serverId          |Number     |M|-|A Server ID of the monitoring server.|
+|url               |URI2047    |M|-|A URL of monitoring server. [[Description](#user-content-servertype)]|
+|type              |string     |M|-|A type of monitoring server. [[List](#user-content-servertype)]|
+|nickName          |String255  |M|-|A nickname of monitoring server.|
+|userName          |String255  |M|-|A user name of monitoring server.|
+|password          |String255  |M|-|A pass word of monitoring server.|
+|pollingIntervalSec|Number     |M|-|Polling interval seconds.|
+|retryIntervalSec  |Number     |M|-|Retry interval seconds in case of fail polling.|
+|extendedInfo      |String32767|M|-|Can store your any information.|
+
+```json
+{
+  "id": 1,
+  "result": {
+    "serverId": 1,
+    "url": "http://example.com:80",
+    "type": "12345678-9abc-def0-1234-567891abcdef",
+    "nickName": "exampleName",
+    "userName": "Admin",
+    "password": "examplePass",
+    "pollingIntervalSec": 30,
+    "retryIntervalSec": 10,
+    "extendedInfo": "exampleExtraInfo"
+  },
+  "jsonrpc": "2.0"
+}
+```
+
+### getLastInfo(method)
+
+Can send the diffarence between sent data and having data currently to use got lastInfo.
```

Comment:
This procedure is used for getting lastInfo which is stored in Hatohol Server before calling put series procedures.
Its lastInfo is used for put series procedures to calculate changes current value from previous value.